transformer_engine/common/__init__.py

```diff
@@ -123,7 +151,25 @@ def _load_nvrtc():
     return ctypes.CDLL(f"libnvrtc.{_get_sys_extension()}", mode=ctypes.RTLD_GLOBAL)
 
 
+def _load_cublas():
+    """Load CUBLAS shared library."""
+
+    # Attempt to locate CUBLAS in Python dist-packages
+    lib_path = glob.glob(
+        os.path.join(
+            sysconfig.get_path("purelib"),
+            f"nvidia/cuda_nvrtc/lib/libnvrtc.{_get_sys_extension()}.*[0-9]",
```

Comment:
Should this be something like:
```suggestion
            f"nvidia/cublas/lib/libcublas.{_get_sys_extension()}.*[0-9]",
```